building/build-debs/homeworld-admin-tools/src/command.py

```diff
@@ -76,5 +79,5 @@ def main_invoke(command, params):
         invoke(params)
         return 0
     except CommandFailedException as e:
-        print(e)
+        print('\x1b[1;31m' + str(e) + '\x1b[0m')
```

Comment:
Could we add a line saying "Command failed" or something? Currently it is not clear that it is an error, and I feel that this clarity should not solely depend on color.

Reply:
I did that, but it seemed overdone in a number of cases, so I removed it. Do you think it's worthwhile anyway?

I mostly got things like:

    $ spire
    commands.
        [...]
    no command
    * * * * COMMAND FAILED * * * *

Though it's possible that there's a better solution.

Reply:
It would be overdone in a number of cases, but there are also a number of cases where it isn't, like "no keyserver found" or "expected kubelet to be ready". It should be sufficient to prepend "command failed: " in front of the error message, to make it clear that it's an error message.